server/xpub-server/entities/manuscript/resolvers/submitManuscript.js

```diff
@@ -30,7 +30,10 @@ async function submitManuscript(_, { data }, { user, ip }) {
   manuscript.applyInput(manuscriptInput)
 
   const notify = new Notification(config)
-  await notify.sendFinalSubmissionEmail(manuscript.getAuthor())
+  await notify.sendFinalSubmissionEmail(
+    manuscript.getAuthor(),
+    manuscript.meta.title,
+  )
```

Comment:
Can we update the test to assert against the email contents?

We can use `.toMatchSnapshot()`